### ISSUE TITLE
[web] CanvasKit - image orientation from EXIF header ignored

### ISSUE THREAD

**Issue by steffenhaak** [flutter/flutter#70097]
## Steps to Reproduce

1. Create the following app:

```
import 'package:flutter/material.dart';

void main() {
  runApp(MyApp());
}

class MyApp extends StatelessWidget {
  // This widget is the root of your application.
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'CanvasKit Image Orientation Bug',
      theme: ThemeData(
        primarySwatch: Colors.blue,
      ),
      home: MainPage(
        title: 'MainPage',
      ),
    );
  }
}

class MainPage extends StatefulWidget {
  MainPage({Key key, this.title}) : super(key: key);

  final String title;

  @override
  _MainPageState createState() => _MainPageState();
}

class _MainPageState extends State<MainPage> {
  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(
        title: Text(widget.title),
      ),
      body: Center(
        child: Image.network(
          'https:///klubraum-playground-assets.storage.googleapis.com/c3f775ca-c9ba-4bcb-95f9-7e65df142f6c.jpg',
        ),
      ),
    );
  }
}

```

2. Run the app in Google Chrome using Dom-Canvas: flutter run -d chrome
3. Run the app in Google Chrome using CanvasKit: flutter run -d chrome --release --dart-define=FLUTTER_WEB_USE_SKIA=true

**Expected results:** 
Both screens should display the image in the correct orientation.
![image](https://user-images.githubusercontent.com/26477639/98544555-7c3feb00-2294-11eb-9397-5bd6a5650341.png)

**Actual results:** 
The CanvasKit implementation ignores the image orientation in the EXIF information of the chosen image

![image](https://user-images.githubusercontent.com/26477639/98544603-882bad00-2294-11eb-82cd-9c435c37e734.png)

<details>
  <summary>Logs</summary>

```
Logs do not contain relevant information.
```

```
[✓] Flutter (Channel beta, 1.23.0-18.1.pre, on Mac OS X 10.15.7 19H15 x86_64, locale de-DE)
 
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
[✓] Xcode - develop for iOS and macOS (Xcode 12.1)
[✓] Chrome - develop for the web
[✓] Android Studio (version 4.1)
[✓] IntelliJ IDEA Community Edition (version 2020.2.3)
[✓] VS Code (version 1.51.0)
[✓] Connected device (3 available)

• No issues found!
```

</details>

**Comment by TahaTesser** [comment]
Hi @steffenhaak 
I see there's an open issue addressing the case you described.
Please follow up on that issue, I'm closing the current one as a duplicate.
duplicate of https://github.com/flutter/flutter/issues/33572
If you disagree, please write in the comments and I will reopen it.
Thank you

**Comment by steffenhaak** [comment]
Hi @TahaTesser, thank you for your quick reply :)

I believe this is bug is quite similar, but actually a different case. Let me try to explain. In the mentioned issue (https://github.com/flutter/flutter/issues/33572), the problem appears in the default web backend (DomCanvas).

In my example, DomCanvas actually looks fine and Chrome is rendering the image in the correct orientation. The bug only occurs when using the CanvasKit web backend. Also, referring to the comment of @ferhatb in the existing issue, images in CanvasKit are always loaded through a CORS ajax request and not by means of the browser native image tag, so I do not believe this bug has to be fixed in Chrome but rather in the CanvasKit implementation.

To sum it up,  is strongly believe, we have a CanvasKit bug here.

**Comment by TahaTesser** [comment]
@steffenhaak
@ferhatb  just added canvaskit label, I believe it would be if we address the whole EXIF issue for Dom or Canvaskit on the original itself 
Thank you
